AndelaOSP/art-dashboard
# Admin can see user assigned to asset 162248110

## What does this PR do?
- Shows the email (or full name) of user that an asset is assigned to

## Description of Task to be completed?
- Admin can see user assigned to asset on asset list page
- Removes category and sub-category columns from assets list table

## How should this be manually tested?
- Go to the assets list page. On the table you should see the email (or full name) of a user that an asset has been assigned to

## What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162248110

## Any background context you want to add?
N/A

## Important notes
N/A

## Packages installed
N/A

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
